### PR TITLE
Csv contact import v1

### PR DESCRIPTION
**Pull request by go8soft**
Task: https://trello.com/c/TvKgM26V/170-csv-contact-import-spike

CSV Contacts import v1. We will use it as a starting point for improvements. 
It is available for the managers.

### CODE REVIEW

File: app/policies/imported_item_policy.rb [hunk 1]
```diff
@@ -0,0 +1,17 @@
+class ImportedItemPolicy < ApplicationPolicy
+  class Scope < Scope
+    def resolve
+      raise 'Cannot determine scope' unless @user.in_role_name?('manager')
```
Comment (rik86future) [inline app/policies/imported_item_policy.rb:4]: @user.in_role_name?('sysadmin')

File: app/policies/imported_item_policy.rb [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+
+      ImportedItem.all
+    end
+  end
+
+  def index?
+    admin?
```
Comment (rik86future) [inline app/policies/imported_item_policy.rb:11]: sysadmin

File: app/policies/imported_item_policy.rb [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+  end
+
+  def create?
+    admin?
```
Comment (rik86future) [inline app/policies/imported_item_policy.rb:15]: sysadmin

File: app/policies/imported_item_policy.rb (hunk 1 above)
Comment (go8soft) [reply app/policies/imported_item_policy.rb]: Thanks!

**Comment by jamiebuckley** [comment]
nice!  I particularly like the ability to jump back to the contact list to see which records were brought in as part of an import

**Pull request merged**